action.yml

```diff
@@ -1,10 +1,25 @@
 name: 'New deployment tool'
 description: 'Deploy your project with ease'
-runs:
-  using: 'docker'
-  image: 'Dockerfile'
 
 inputs:
   github_token:
     description: 'GITHUB_TOKEN or a repo scoped PAT.'
-    default: ${{ github.token }}
\ No newline at end of file
+    default: ${{ github.token }}
+  version:
+    description: 'Version of the CLI to use.'
+    default: 'main' # using branch name for now for easier deployments 
```

Comment:
when this tool gets to the point where we are deploying alpha or beta versions, this value will need to be modified before we make a tag. For now, pointing to main works for development.